doc/042_Devel_Image.md

```diff
@@ -68,8 +68,11 @@ Go to `docker_development/image_setup/02_devel_image` and add installation comma
 The Dockerfile is intended to contain any OS dependencies the workspace has (that are not already provided by the base image).
 
 Especially if you are not sure which packages need to be installed at this point, it might make sense to start with the next step, to prepare the workspace. Then you can complete the dependencies in the Dockerfile, using the helper scripts for dependency enumeration.
-The `list_*_osdeps.*` scripts help to determine which workspace-specific packages are required by the workspace.
-These are available for Rock and ROS workspaces and list all OS dependencies depending on the packages' provided metadata. They can be found in the `image_setup/02_devel_image` folder and the output format allows easy copy/pasting into the Dockerfile.
+
+You can start `bash /opt/write_osdeps.bash` scripts help to determine which workspace-specific packages are required by the workspace.
+The script detects Rock and ROS workspaces and list all OS dependencies depending on the packages' provided metadata about dependencies is available.
+The workspace dependencies are written into the `image_setup/02_devel_image/workspace_os_dependencies.txt` and install upon buildding the docker image.
+Install instructions for packages that are not workspace dependencies (or in the metadata) sould be directly added into the dockerfile
```

Comment:
`Install instructions for packages that are not workspace dependencies (or in the metadata) sould be directly added into the dockerfile`

Directly into the dockerfile or by using `workspace_os_dependencies.txt`?

I think to avoid confusion, both ways should work. Also adding write_osdeps.bash as a startscript would be handy imo :+1:

Reply:
we could just add /opt/ to the PATH? then also setup workspace, etc would work that way

Reply:
added to PATH using the Dockerfile

Reply:
and removed it again, you can call it anyway by using `./exec /opt/write_osedps.bash`